SpecialOperations2.Altis/sos/logistics/functions/fn_spawnBox.sqf

```diff
@@ -0,0 +1,57 @@
+/*
+ * Author: EnquiringStone [S.O.S. Captain]
+ * Spawns a box in a specified area.
+ * Thanks: Legman [S.O.S. Mayor]
+ *
+ * Arguments:
+ * 0: name <STRING>
+ * 1: position <ARRAY>
+ * 2: radius <NUMBER>
+ * 3: direction <NUMBER>
+ * 4: contents <ARRAY>
+ *
+ * Return Value:
+ * Nothing
+ *
+ * Example:
+ * [_name, _position, _radius, _direction, _contents] call SOS_fnc_spawnBox;
+ *
+ */
+ 
+if (isDedicated) exitWith {};
+waitUntil {!isNull player};
+
+private ["_name", "_position", "_radius", "_direction", "_contents", "_box"];
+
+_name           = _this select 0;
+_position       = _this select 1;
+_radius         = _this select 2;
+_direction      = _this select 3;
+_contents       = _this select 4;
+
+_box = createVehicle [_name, _position, [], _radius, "NONE"];
```

Comment:
Rename ALL instances of **box** -> **crate**

Function should be **SOS_fnc_spawnCrate**